timemachine/fe/free_energy.py

```diff
@@ -728,6 +726,7 @@ def run_sims_sequential(
     # u_kln matrix (2, 2, n_frames) for each pair of adjacent lambda windows and energy term
     u_kln_by_component_by_lambda = []
 
+    unbound_impls = [p.potential.to_gpu(np.float32).unbound_impl for p in initial_states[0].potentials]
```

Comment:
Would it be worth adding a warning that this assumes all initial states have the same potentials (as [here](https://github.com/proteneer/timemachine/blob/7f1bffc7f24fd1d167b43608f1f1a0f6ae1a07fa/timemachine/fe/free_energy.py#L986))?